### DISCUSSION TITLE
Forwarding discord conversation - Tramadol combination suggestion

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#151] [Combination Chart]
This is a forwarding of a discord suggestion by user "Time" (murumassa) - https://discord.com/channels/179641883222474752/1237236916181667892/1237236917809188946

"Hey, I've pretty much finished this write up I've been working on, it coveres tram x amph, tram x meth, tram x mdma. I'm pretty confident with the findings for seizures however am more unsure about SS. My findings SEEM to indicate that there is an increased risk of SS with meth and mdma and lesser so with amph however I've seen conflicting information and am not 100% about how different serotonin agonists and reuptake inhibiters interact. Tram is an sri and a nri. Meth and MDMA both cause an increased release of serotonin. To me this seems like it would increase the risk of SS.

[they referenced a discord forum question discussing this]
I wasn't able to find much information directly speaking on any of these combos, which made this quite challenging so most of the drawn conclusions come from compounding effects for example both drugs lower the seizure threshold

My current writeup for context (it currently says that there is an increased risk of SS but if my understanding is wrong I'll obviously change it"

They then uploaded a PDF discussing their findings and proposal for revisions
[An_Analysis_of_Combining_Tramadol_with_Methamphetamine_Amphetamine_and_34-Methylenedioxymethamphetamine.pdf](https://github.com/TripSit/drugs/files/15244868/An_Analysis_of_Combining_Tramadol_with_Methamphetamine_Amphetamine_and_34-Methylenedioxymethamphetamine.pdf)

**Comment by TimeX4** [comment]
Hello! I'm uploading an updated version of the PDF. This has just been edited, there aren't any content differences. 

[An Analysis of the Combination of Tramadol with Methamphetamine, Amphetamine, and 3,4-Methylenedioxymethamphetamine.pdf](https://github.com/TripSit/drugs/files/15251806/An.Analysis.of.the.Combination.of.Tramadol.with.Methamphetamine.Amphetamine.and.3.4-Methylenedioxymethamphetamine.pdf)

**Comment by TimeX4** [comment]
Update - Tramadol x MDMA

Low / No Risk of SS 

https://link.springer.com/article/10.1007/s00213-022-06083-y
https://link.springer.com/article/10.1007/s00213-021-05876-x

Still investigating methamphetamine. 

Will post a final document soon :)

**Reply by utaninja** [reply]
I had most of my response written out already so I've posted it below. I do not see how either of these studies indicate there is no/low serotonin toxicity concerns with MDMA. Please can you quote/show me directly where this is indicated. In reference to Methamphetamine as I noted below, I'll need to wait on that when I have time.

**Comment by utaninja** [comment]
Hey there. As noted on discord, I really appreciate all of the work you put into this. I've broken this done and been as thorough as possible, please let me know if I missed anything!

Okay, so breaking down my thoughts-

It seems that you are proposing a modification to the current interactions on the combination chart (Tramadol + Amphetamines and Tramadol + MDMA)
In addition, you are proposing combinations for amphetamine, methamphetamine, and MDMA.

The current combination chart says the following:
Tramadol + Amphetamines = Dangerous - Tramadol and stimulants both increase the risk of seizures.
Tramadol + MDMA = Dangerous - Tramadol and stimulants both increase the risk of seizures.

Just to note, the reason the TripSit bot doesn’t have “Amphetamines” (tho you can force it to do this but it doesn’t pop up) is because they have single drugs only. The combination chart currently has many drug categories (such as 2C-x and Amphetamines) however there is talk of changing this, you can see this here: https://github.com/TripSit/drugs/discussions/114

I’m going to focus on the combination chart effects and then when I get the chance I can review for methamphetamine and amphetamine (the individual drugs)

Just to derive from your suggestion it seems you would be suggesting-

Tramadol + Amphetamines
Change to note and addition of sources. - “There is significant evidence to support increased risk of seizure. There is also some evidence to support a minor increase in risk of serotonin syndrome.”

Tramadol + MDMA
Change to note and addition to sources. - “There is significant evidence to support increased risk of seizure, this risk is further increased if hyponatraemia is present. Additionally, there is evidence to support increased risk of developing serotonin syndrome.”

Tramadol + Amphetamines
---

Tramadol + Amphetamines
Change to note and addition of sources. - “There is significant evidence to support increased risk of seizure. There is also some evidence to support a minor increase in risk of serotonin syndrome.”

Bowyer, J. F., Thomas, M., Schmued, L. C., & Ali, S. F. (2008). Brain Region–Specific Neurodegenerative Profiles Showing the Relative Importance of Amphetamine Dose, Hyperthermia, Seizures, and the Blood–Brain Barrier. Annals of the New York Academy of Sciences, 1139(1), 127–139. https://doi.org/10.1196/annals.1432.005
* Great study shows amph has seizure concerns from what I can see

Riffee, W. H., & Gerald, M. C. (1977). The effects of chronic administration and withdrawal of (+)- amphetamine on seizure threshold and endogenous catecholamine concentrations and their rates of biosynthesis in mice. Psychopharmacology, 51(2), 175–179. https://doi.org/10.1007/BF00431736
* Adds to this

““Amphetamines/Cannabinoids/Tramadol Interaction/Withdrawal,” 2019”
I’m unable to access this, however I’m hesitant to include this as it has another drug involved and seems to have external variables and limited information.

“http://www.ncbi.nlm.nih.gov/books/NBK556103/”
* I'd like this to be expanded much more-
I don’t see any reports or studies showing that there is a risk that should be considered when it comes to amphetamine raising the risk of serotonin syndrome. I’d also like to note that in multiple of the studies you linked, they discussed interactions but as far as I could tell never noted amphetamine/similar. (Example: Food and Drug Administration. (2022). TRAMADOL HYDROCHLORIDE. Food and Drug Administration. https://www.accessdata.fda.gov/spl/data/77901cc0-3753-4636-89c7- 86b1fe743dad/77901cc0-3753-4636-89c7-86b1fe743dad.xml#section-7)

I did find this study that talked about amphetamine regarding serotonin however, I’d hope to see more regarding this. Do you happen to have any other sources regarding that?
https://pubmed.ncbi.nlm.nih.gov/9109529/#:~:text=Amphetamine%20increases%20extracellular%20norepinephrine%20and,effects%20of%20amphetamine%2Dlike%20stimulants.

Regarding Tramadol and Amphetamines for seizures, can leave the current interaction as is and list the Tramadol/+Amphetamine papers that you sent for this.

(Tramadol sources that I saw):
Gardner, J. S., Blough, D., Drinkard, C. R., Shatin, D., Anderson, G., Graham, D., & Alderfer, R. (2000). Tramadol and Seizures: A Surveillance Study in a Managed Care Population. Pharmacotherapy: The Journal of Human Pharmacology and Drug Therapy, 20(12), 1423–1431. https://doi.org/10.1592/phco.20.19.1423.34854

Jovanović-Čupić, V., Martinović, Ž., & Nešić, N. (2006). Seizures Associated with Intoxication and Abuse of Tramadol. Clinical Toxicology, 44(2), 143–146. https://doi.org/10.1080/1556365050014418

I’d also like to add this study that I found- Talaie, H., Panahandeh, R., Fayaznouri, M. R., Asadi, Z., & Abdollahi, M. (2009). Dose-independent occurrence of seizure with tramadol. Journal of Medical Toxicology, 5(2), 63–67. https://doi.org/10.1007/BF03161089

Sansone, R. A., & Sansone, L. A. (2009). Tramadol. Psychiatry (Edgmont), 6(4), 17–21. Thomas, D. M., Pérez, M. A., Francescutti-Verbeem, D. M., Shah, M. M., & Kuhn, D. M. (2010). 
* This does not seem to contribute much to what has already been provided. Do you see something that stands out here?

Tramadol + MDMA
-
Tramadol + MDMA
Change to note and addition to sources. - “There is significant evidence to support increased risk of seizure, this risk is further increased if hyponatraemia is present. Additionally, there is evidence to support increased risk of developing serotonin syndrome.”

These studies look good to me-
Giorgi, F. S., Pizzanelli, C., Ferrucci, M., Lazzeri, G., Faetti, M., Giusiani, M., Pontarelli, F., Busceti, C. L., Murri, L., & Fornai, F. (2005). Previous exposure to (±) 3,4- methylenedioxymethamphetamine produces long-lasting alteration in limbic brain excitability measured by electroencephalogram spectrum analysis, brain metabolism and seizure susceptibility. Neuroscience, 136(1), 43–53. https://doi.org/10.1016/j.neuroscience.2005.07.036

Hartung, T. K., Schofield, E., Short, A. I., Parr, M. J. A., & Henry, J. A. (2002). Hyponatraemic states following 3,4‐methylenedioxymethamphetamine (MDMA, ‘ecstasy’) ingestion. QJM: An International Journal of Medicine, 95(7), 431–437. https://doi.org/10.1093/qjmed/95.7.43

I’d like to contribute this source aswell for a general overview on MDMA and it’s serotonin toxicity concerns, which in combination with the Tramadol sources noted above, should be good for the serotonin syndrome concern.

Parrott, A. C. (2002). Recreational Ecstasy/MDMA, the serotonin syndrome, and serotonergic neurotoxicity. Pharmacology Biochemistry and Behavior, 71(4), 837–844. https://doi.org/10.1016/S0091-3057(01)00711-0
Additionally, the Tramadol sources discussing seizure concerns in combination with the MDMA seizure related studies below should be good for that regard.

To review
---
There are sources that you have provided for sources for Tramadol + Amphetamine in regards to seizures. I do not believe there has been enough to show a concern for anything further than this. For Tramadol + MDMA there are sources that you have provided for seizure concerns and these have been noted, there also seems to be an indication to have a additional note both for the dehydration concern of contributing to seizures ~~and the concern of serotonin syndrome for which there seems to be enough evidence for~~. 

ps: I would take a look around the GitHub, especially some of our issues and pull requests, you can see generally what sort of studies/things we are looking for. We also discuss stuff in the read-me. really appreciate the help!

**Reply by TimeX4** [reply]
I'm going to use this reply as the "formal" post I originally intended to submit once I was finished with the document (which I think I am now), and then also reply to the points you made. 

___

### Why?
I chose these specific combinations as I have found discrepancies between the combo chart and the tripsit bot or missing data for each of the three interactions.

**TRAM x AMPH (Tripbot):** There is currently **NO** tripbot entry for TRAM x AMPH.
**TRAM x METH (Tripbot):** There is currently **NO** tripbot entry for TRAM x METH.
**TRAM x MDMA (Tripbot):** Tripbot currently suggests **Low Risk & No Synergy** with regard to TRAM x METH
![image](https://github.com/TripSit/drugs/assets/75556349/26cae70a-07ea-4dd5-a5b9-665375e06619)
**TRAM x MDMA (Combo):** The combo chart currently lists this combonation as **Dangerous** and states "Tramadol and stimulants both increase the risk of seizures." (The same message it shows for amphetamines).
This is the discrepancy I originally noticed and that sent me down this rabbit hole. 

___

### Tramadol x Amphetamine*s* (as a class)

**TRAM x AMPHs (Combo Chart):** I understand this is referring to amphetamines as a group, not specifically to the drug amphetamine. I don't believe there are any issues with what it currently says.  
**TRAM x AMPHs (Tripbot):** When forcing the bot to give an answer for amphetamines as a class it shows the following:
![image](https://github.com/TripSit/drugs/assets/75556349/bedebd26-0cfb-4a14-a634-81f1b06614ec)
This is **NOT** the same as what the combo chart says. My paper only covers the specific drugs but if you think the seizure findings of amphetamine are generally applicable to amphetamines as a class then I would suggest this be changed.

___

### My Findings (The important section) 

I will post a relevant excerpt from my paper that highlights the findings. 

This paper finds that when tramadol is mixed with:

1. Methamphetamine – There is significant evidence to support increased risk of seizure and increased risk of developing serotonin syndrome. There is also evidence of increased hyperlocomotion. 
2. Amphetamine – There is significant evidence to support increased risk of seizure. 
3. 3,4-Methylenedioxymethamphetamine – There is significant evidence to support increased risk of seizure, this risk is further increased if hyponatraemia is present.

### Sources & Research

Everything can be found in my paper. _Note to utaninja: I know you spent a lot of your valuable time reading this paper already, the only thing that has changed since last time is with regard to meth and mdma and serotonin syndrome._
[An Analysis of the Combination of Tramadol with Methamphetamine, Amphetamine, and 3,4-Methylenedioxymethamphetamine.pdf](https://github.com/TripSit/drugs/files/15287099/An.Analysis.of.the.Combination.of.Tramadol.with.Methamphetamine.Amphetamine.and.3.4-Methylenedioxymethamphetamine.pdf)

___

### Responding to Utaninja

> It seems that you are proposing a modification to the current interactions on the combination chart (Tramadol + Amphetamines and Tramadol + MDMA)
In addition, you are proposing combinations for amphetamine, methamphetamine, and MDMA.

Correct except for the TRAM x AMPHs combo, I'm not if the entire class is in the scope of my findings, I'll let you be the judge of that.

> I’m unable to access this, however I’m hesitant to include this as it has another drug involved and seems to have external variables and limited information.

Nothing of note really, just some case study that was fascinating, I didn't use anything it concluded in the paper. Kinda a further reading type thing. 

> I don’t see any reports or studies showing that there is a risk that should be considered when it comes to amphetamine raising the risk of serotonin syndrome.

I agree, it has been removed from the final version, in theory, any serotonin being released could increase the risk but the amounts are low and I don't think it has any significance. 

> I did find this study that talked about amphetamine regarding serotonin however, I’d hope to see more regarding this. Do you happen to have any other sources regarding that?

I do not unfortunately, my understanding is that the amount of serotonin released by AMPH is very low especially when compared to dopamine and norepinephrine and thus isn't a primary interest of research; part of the reason this was difficult to write haha.

> This does not seem to contribute much to what has already been provided. Do you see something that stands out here?

Nope, I only used this to cite TRAM as an SNRI in the introduction.

___

### With regard to https://github.com/TripSit/drugs/discussions/151#discussioncomment-9405462

The relevant sections are quoted in the final paper but I'll send the full sections here. 

From [Malcolm and Thomas, 2022](https://link.springer.com/article/10.1007/s00213-021-05876-x)
> **Serotonin reuptake inhibitors with MDMA (SRI + SRA)**
> 
> Despite SRIs and MDMA (SRA) both being able to increase intrasynaptic 5HT, several small clinical trials combining them suggest that it is usually a benign combination (Fig. [2](https://link.springer.com/article/10.1007/s00213-021-05876-x#Fig2)); it is been documented that while SRA’s inhibition of vesicular monoamine transporter 2 (VMAT2) disrupts vesicle packaging and leads to carrier-mediated 5-HT release via SERT, SRIs competition at the serotonin reuptake transporter (SERT) results in attenuated subjective and physical effects of the SRA (Gudelsky and Nash [1996](https://link.springer.com/article/10.1007/s00213-021-05876-x#ref-CR33); Hysek [2012](https://link.springer.com/article/10.1007/s00213-021-05876-x#ref-CR36); Stein and Rink [1999](https://link.springer.com/article/10.1007/s00213-021-05876-x#ref-CR73); Tancer and Johanson [2007](https://link.springer.com/article/10.1007/s00213-021-05876-x#ref-CR76); Farre [2007](https://link.springer.com/article/10.1007/s00213-021-05876-x#ref-CR20); Liechti and Vollenweider [2000](https://link.springer.com/article/10.1007/s00213-021-05876-x#ref-CR43)). These attenuated effects may also translate to decreased therapeutic benefit since efficacy response for PTSD was lower for the study participants who recently tapered off of an antidepressant at least 2 weeks prior to MDMA-assisted therapy (mean 25.1 days) (Feduccia et al. [2021](https://link.springer.com/article/10.1007/s00213-021-05876-x#ref-CR21)).
> 

From [Sarparast et al., 2022](https://link.springer.com/article/10.1007/s00213-022-06083-y)

> The RCTs in this review show attenuation of MDMA’s physiological effects, suggesting a reduced odds of serotonin syndrome, at least with acute SSRI dosing. This is corroborated by at least one other scientific review positing SSRIs, SNRIs, and TCAs are not likely to increase serotonin to life-threatening toxicity when combined with MDMA (Silins et al. [2007](https://link.springer.com/article/10.1007/s00213-022-06083-y#ref-CR127)).

**Reply by utaninja** [reply]
Thank you for the updated formatting and clarification. This stuff gets pretty confusing and I appreciate your patience.

I agree with Tramadol + MDMA's modification and sources. I believe there is enough evidence to support Tramadol + Amphetamine for Tramadol + Amphetamines modification that I proposed. (also noting that this is what the chart currently says, while it does not have sources, it was reviewed many times from my understanding)

This would also transfer over to Tramadol + MDMA for the TripBot (as it pulls from the source) and I'm in favor of the suggestion for Tramadol + Amphetamine (TripBot).

I will review the suggestion for Tramadol + Methamphetamine soon.

@LunaUrsa -
"^TRAM x AMPHs (Tripbot): When forcing the bot to give an answer for amphetamines as a class it shows the following:"
Why would this be shown for the answer for this combination (when manually inputted for amphetamines) shouldn't it show what is in combos.json/the combination chart?

**Comment by utaninja** [comment]
Will respond soon, but I wanted to note this PR. https://github.com/TripSit/drugs/pull/19
Looks like the interaction has been updated (the site isn't updated for it yet)

},
    "amphetamines": {
      "note": "This combination of stimulants will increase strain on the heart, may cause some physical discomfort, and has the chance to cause cardiovascular issues. The anxiogenic and focusing effects of stimulants can increase the chance of unpleasant thought loops and make the experience more uncomfortable, this combination raises these chances. Amphetamines will increase the neurotoxic effects of MDMA, in addition to causing further concerns of hyperthermia due to the inherent nature of the combination. It will also raise one's body temperature, also likely making the combination more neurotoxic.",
      "sources": [
        {
          "author": "Kelly J. Clemens, Iain S. McGregor, Glenn E. Hunt, Jennifer L. Cornish",
          "title": "MDMA, methamphetamine and their combination: possible lessons for party drug users from recent preclinical research",
          "url": "https://doi.org/10.1080/09595230601036945"
        },
        {
          "author": "Kelly J. Clemens, Iain S. McGregor, Glenn E. Hunt, Jennifer L. Cornish",
          "title": "Repeated weekly exposure to MDMA, methamphetamine or their combination: long-term behavioural and neurochemical effects in rats ",
          "url": "https://doi.org/10.1016/j.drugalcdep.2006.06.004"
        },
        {
          "author": "P. Leon Brown, Eugene A. Kiyatkin",
          "title": "Brain hyperthermia induced by MDMA (‘ecstasy’): modulation by environmental conditions",
          "url": "https://doi.org/10.1111/j.0953-816X.2004.03453.x"
        }

**Reply by TimeX4** [reply]
This is MDMA + AMPH I didn't cover this at all.

**Reply by utaninja** [reply]
Right. To clarify I meant that it might share some resources that may be helpful for Tramadol + Amph/x and Tramadol + MDMA

**Reply by TimeX4** [reply]
Ah I see, I misunderstood. Thank you for sharing.

**Comment by utaninja** [comment]
@TimeX4 sorry this took so long! It must've gotten buried and slipped my mind. Thanks for all the hard work you put in this.

All three interaction suggestions have been pushed:
Tramadol + Amphetamines https://github.com/TripSit/drugs/issues/333
Tramadol + MDMA https://github.com/TripSit/drugs/issues/334
Tramadol + Methamphetamine https://github.com/TripSit/drugs/issues/335
